docs/variants/novacustom_nv4x/overview.md

```diff
@@ -40,9 +40,9 @@ in the [release page](releases.md).
     validation procedure.
 - [Test matrix](test-matrix.md) - describes validation scope used during
     Dasharo firmware validation procedure.
-- [Post-installation setup](post_install.md) - describes the post installation
-    steps which should be performed to unlock the full functionality of the
-    product.
+- [Post-installation setup](/common-coreboot-docs/clevo_post_install/) -
+    describes the post installation steps which should be performed to unlock
+    the full functionality of the product.
 - [Updating firmware using fwupd](/common-coreboot-docs/fwupd_usage/) -
     describes the method of updating the Dasharo firmware by using fwupd daemon.
 - [Logo customization](/common-coreboot-docs/custom_logo.md) - describes the
```

Comment:
These three references don't work after building the site locally.

Reply:
@Pokisiekk now, all links should work properly